braindecode/models/eegsimpleconv.py

```diff
@@ -0,0 +1,148 @@
+"""
+EEG Simple Convolutional Neural Network from Yassine El Ouahidi et al. (2023).
+
+Each EEG channels is convoled individually.
+
+"""
+# Authors: Yassine El Ouahidi <eloua.yas@gmail.com>
+#
+# License: BSD-3
+
+import torch
+
+from . import Resample
+from .base import EEGModuleMixin
+
+
+class EEGSimpleConv(EEGModuleMixin, torch.nn.Module):
+    """EEGSimpleConv from [Yassine2023]_.
+
+    Can you increase the description here Yassine?
+
+    The paper and original code with more details about the methodological
+    choices are available at the [Yassine2023]_ and [Yassine2023Code]_.
+
+    Notes
+    -----
+    The authors recommend using the default parameters for MI decoding.
+    Please refer to the original paper and code for more details.
+
+    An intesive ablation study is available in the paper to understand the
+    impact of each parameter on the model performance.
+
+    .. versionadded:: 0.9
+
+    Parameters
+    ----------
+    fm: int
+        Number of Feature Maps at the first Convolution, width of the model.
+    n_convs: int
+        Number of blocks of convolutions (2 convolutions per block), depth of the model.
+    resampling: int
+        Resampling Frequency.
+    kernel_size: int
+        Size of the convolutions kernels.
+
+    References
+    ----------
+    .. [Yassine2023] Yassine El Ouahidi, V. Gripon, B. Pasdeloup, G. Bouallegue
+    N. Farrugia, G. Lioi, 2023. A Strong and Simple Deep Learning Baseline for
+    BCI Motor Imagery Decoding. Arxiv preprint. arxiv.org/abs/2309.07159
+    .. [Yassine2023] Yassine El Ouahidi, V. Gripon, B. Pasdeloup, G. Bouallegue
+    N. Farrugia, G. Lioi, 2023. A Strong and Simple Deep Learning Baseline for
+    BCI Motor Imagery Decoding. GitHub repository.
+    https://github.com/elouayas/EEGSimpleConv.
+
+    """
+
+    def __init__(
+        self,
+        # Base arguments
+        n_outputs=None,
+        n_chans=None,
+        sfreq=None,
+        # Model specific arguments
+        fm=128,
```

Comment:
Can you put a more descriptive name for fm?

Reply:
Done !